Events/src/main/java/com/github/swcraftlyon/meetup/infra/dao/MangoEventDao.java

```diff
@@ -0,0 +1,41 @@
+package com.github.swcraftlyon.meetup.infra.dao;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+
+public class MangoEventDao implements IEventDAO {
+
+    public static final String EVENT_DB = "EventDB";
+    public static final String MONGODB_LOCALHOST_27017 = "mongodb://localhost:27017";
```

Comment:
You mean that i create a ApplicationConfig.java that contains constants of my mongodb server parameters ? then i call this class in my dao ? i will create a dependency between my DAO and my ApplicationConfig ? Do you have any suggestions please ?